PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,12 +10,9 @@
 
 ### Before Merging
 
-- [ ] If there is an RC on this branch, revert the version change in `version.rb`
+- [ ] Bump the version in `version.rb` following [SemVer](https://semver.org) **on this branch** (if you were testing an RC, make sure you remove the RC before merging)
```

Comment:
I'm wondering if this should be a post-merge step?  In my mind, and especially if we ever had open source contributions to this repo, we'd want the ability to batch up changes and release them out-of-band of approving PRs.  What do you think?

Reply:
That's an interesting question I hadn't considered! I think the answer depends on whether or not you and @brettfishman (as the code owners 😄) want to more explicitly manage the release process and batch changes together.

My perspective is that this repo doesn't get a ton of activity, so changes may be too few and far between to batch. I also like the idea of putting the ownness on the person who opens the PR to get it merged _and_ released. And that's easier to do when they only have to worry about their PR as a single "release" and not coordinate with other work that may be in flight. From what I've seen with our internal Ruby gems (caveat is that I haven't worked on open source much), we treat the PR approval as "this is code that is good to release and should be available for apps to use immediately". So, if that's the same sentiment we want to have here, I think it makes sense to do the version bump with the PR itself.

That's just my opinion, though, and I'm totally fine with it being overridden if others feel strongly!

Reply:
You can TOTALLY ignore me at this point, but…I agree with Cameron.  It's generally simpler to have a single way of thinking about release management, and for us that's "changes get released when they are read" i.e. continuous delivery.

Reply:
Sounds like some well thought out opinions.  I am swayed.

Reply:
I agree with this as well. Thanks @cameronjacoby for making this change!